components/CMakeLists.txt

```diff
@@ -2,22 +2,26 @@ project (Components)
 
 # Version file
 set (VERSION_IN_FILE "${OpenMW_SOURCE_DIR}/files/version.in")
-set (VERSION_FILE "${OpenMW_BINARY_DIR}/resources/version")
+set (VERSION_FILE_PATH_BASE "${OpenMW_BINARY_DIR}")
+set (VERSION_FILE_PATH_RELATIVE resources/version)
 if (GIT_CHECKOUT)
   add_custom_target (git-version
        COMMAND ${CMAKE_COMMAND}
             -DGIT_EXECUTABLE=${GIT_EXECUTABLE}
             -DPROJECT_SOURCE_DIR=${PROJECT_SOURCE_DIR}
             -DVERSION_IN_FILE=${VERSION_IN_FILE}
-            -DVERSION_FILE=${VERSION_FILE}
+            -DVERSION_FILE_PATH_BASE=${VERSION_FILE_PATH_BASE}
+            -DVERSION_FILE_PATH_RELATIVE=${VERSION_FILE_PATH_RELATIVE}
             -DOPENMW_VERSION_MAJOR=${OPENMW_VERSION_MAJOR}
             -DOPENMW_VERSION_MINOR=${OPENMW_VERSION_MINOR}
             -DOPENMW_VERSION_RELEASE=${OPENMW_VERSION_RELEASE}
             -DOPENMW_VERSION=${OPENMW_VERSION}
+            -DMSVC=${MSVC}
```

Comment:
`MSVC=` not needed anymore, is it?

Reply:
If the current build passes CI, then it can go, but I've got a sneaking suspicion that it won't.